TypeSystem/Schema/serialized/smoke_test_struct.json

```diff
@@ -40,10 +40,22 @@
   ],
   "o": null,
   "q": {
-    "B2": {
-      "a": 1073741824
+    "C": {
```

Comment:
I assigned a non-default Variant value to "q" to test the nested ones (more use cases now covered). See the test.cc change.